src/ripple/app/paths/PathRequest.cpp

```diff
@@ -175,70 +166,69 @@ void PathRequest::updateComplete ()
 bool PathRequest::isValid (RippleLineCache::ref crCache)
 {
     ScopedLockType sl (mLock);
-    bValid = raSrcAccount && raDstAccount &&
-            saDstAmount > zero;
-    auto const& lrLedger = crCache->getLedger ();
+    if (! raSrcAccount || ! raDstAccount)
+        return false;
 
-    if (bValid)
+    if (! convert_all_ && (saSendMax || saDstAmount <= zero))
```

Comment:
`convert_all_` should probably be camel case.  Honestly, I'd name it in the same naming convention as the rest of the class, `bConvertAll`.  I hate Polish notation, but I like consistency.